src/Generator.Bind/XML/ParsingHelpers.cs

```diff
@@ -163,7 +163,7 @@ public static TypeSignature ParseTypeSignature([NotNull] string type)
                 var firstPointerLevelIndex = typeName.IndexOf('*');
                 var lastPointerLevelIndex = typeName.LastIndexOf('*');
 
-                pointerLevel = Math.Abs(lastPointerLevelIndex - firstPointerLevelIndex) + 1;
+                pointerLevel = System.Math.Abs(lastPointerLevelIndex - firstPointerLevelIndex) + 1;
```

Comment:
Is this required? I can't think of a reason the binder uses the Math subsystem.

Reply:
Apologies, was the result of find+replace